commands/host.vegman

```diff
@@ -193,35 +238,64 @@ function cmd_virtualmedia_mount {
     return 1
   fi
 
-  if [[ ! -e ${imagefile} ]] || [[ -d ${imagefile} ]]; then
-    echo "Please specify the path to image file" >&2
+  if ! [[ ${imagefile} =~ ${imagefile_pattern} ]]; then
+    echo "Please specify the valid path to the image file" >&2
     return 1
   fi
 
-  usb-ctrl insert "usb.ms.cli" "${imagefile}" "${type}"
+  if [[ ! -z ${username} ]] || [[ ! -z ${password} ]]
+  then
+    printf %b "${username:0:64}\0${password:0:64}\0" >6
+    cred_pipe=6
+  fi
+
+  $(busctl call xyz.openbmc_project.VirtualMedia \
+    /xyz/openbmc_project/VirtualMedia/Legacy/USB1 \
+    xyz.openbmc_project.VirtualMedia.Legacy \
+    Mount sbyv "$imagefile" $readonly ${type_id} y $cred_pipe  >/dev/null) && \
+  echo "The USB1 Virtual Media Device mounted"
```

Comment:
Now. the system has the `virtual-media` service, which defines each `VM-endpoint` as `DBus` object.
The VEGMAN system has a tow VM-endpoints: WebUI Proxy (USB0) and Legacy(USB1).
The USB0 can be mounted only via WebUI because it uses the NBD-proxy-server. In the CLI we can only get the status of the `USB0` and unmount it.
For the USB1 we possible to `mount`/`umount`/`status` via `CLI`

Reply:
I'd like you to mention this somewhere in help message.